Italian/smartphone/facebook/facebook.md

```diff
@@ -1,2 +1,7 @@
-Accidenti!
-Ho un milione di amici ma non riconosco nessuno.
\ No newline at end of file
+Accidenti!  
+Ho un milione di amici ma non riconosco nessuno.  
+
+La situazione non è delle più rosee, ma devi pur far qualcosa, quindi decidi di
+
+contattare [un amico a caso](qualsiasi/qualsiasi.md)   
+contattare [il primo amico nella lista](primo/primo.md)
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.